GameData/ROCapsules/PartConfigs/Vostok/VostokCapsule.cfg

```diff
@@ -1,193 +1,203 @@
-	
-	TechRequired = unlockParts
-	cost = 150
-	entryCost = 1
-	
+
 	vesselType = Ship
 	stagingIcon = COMMAND_POD
 	CrewCapacity = 1
-	
-	INTERNAL
-	{
-		name = SoyuzPod
-	}
-	
-	//  ============================================================================
+	bulkheadProfiles = size1, size2
+
+	//	============================================================================
 	//	Thermo, Durability
-	//  ============================================================================
-		
+	//	============================================================================
+
+	mass = 2.325
+	dragModelType = default
+	maximum_drag = 0.2
+	minimum_drag = 0.2
+	angularDrag = 2
 	crashTolerance = 14
-	maxTemp = 2400
-	skinMaxTemp = 3600
-	emissiveConstant = 0.4 // not too absorptive for reentry
+	breakingForce = 250
+	breakingTorque = 250
+	fuelCrossFeed = True
+
+	heatShieldDiameter	= 2.3
+	heatShieldTag = LEO
+	resetHeatShieldAblator = true
+	resetHeatShieldMass = false
+	emissiveConstant = 0.6
 	thermalMassModifier = 1.0
 	skinMassPerArea = 4
-	
-	MODULE:NEEDS[!DeadlyReentry]
-	{
-		name = ModuleAblator
-		ablativeResource = Ablator
-		lossExp = -8000
-		lossConst = 0.1
-		pyrolysisLossFactor = 25000
-		ablationTempThresh = 500
-		reentryConductivity = 0.01
-	}
-	
-	MODULE:NEEDS[DeadlyReentry]
-	{
-		name = ModuleHeatShield
-		ablativeResource = Ablator
-		lossExp = -8000
-		lossConst = 0.1
-		pyrolysisLossFactor = 25000
-		ablationTempThresh = 500
-		reentryConductivity = 0.01
-		depletedMaxTemp = 1200
-	}
-	
-	RESOURCE
-	{
-		name = Ablator
-		amount = 225
-		maxAmount = 225
-	}
-	
-	//  ============================================================================
+
+	//	============================================================================
 	//	Modules and Resources
-	//  ============================================================================
-	
+	//	============================================================================
+
 	MODULE
 	{
 		name = ModuleCommand
 		minimumCrew = 0
 		RESOURCE
 		{
 			name = ElectricCharge
-			rate = 0.25 // A little more than half the Mercury Capsule as we will put it in the SM
+			rate = 0.25
 		}
 	}
-	
-	// Cannot EVA from Here
-	MODULE:NEEDS[RP-0]
+
+	MODULE
 	{
-		name = ModuleNoEVA
+		name = ModuleSAS
+		SASServiceLevel = 3
 	}
-	
+
 	MODULE
 	{
 		name = ModuleScienceContainer
 		reviewActionName = Review Stored Data
 		storeActionName = Store Experiments
 		storageRange = 3.0
-	} 
-	
+	}
+
 	MODULE
 	{
-		name = TacGenericConverter
-		converterName = CO2 Scrubber
-		StartActionName = Start CO2 Scrubber
-		StopActionName = Stop CO2 Scrubber
-		tag = Life Support
-		GeneratesHeat = False
-		UseSpecialistBonus = True
-		SpecialistEfficiencyFactor = 0.2
-		SpecialistBonusBase = 0.05
-		ExperienceEffect = ConverterSkill
-		EfficiencyBonus = 1
-		conversionRate = 1.0	// # of people - Figures based on per/person
-
-		INPUT_RESOURCE
+		name = ModuleFuelTanks
+		volume = 1
```

Comment:
Include volume from the TACLS portion.  Keep a common tank size, let the player fill it as needed